apps/dashboard/src/utils/betaUitl.ts

```diff
@@ -0,0 +1,4 @@
+export function isBetaEnabled(): boolean {
+    const match = document.cookie.match(/(?:^|;\s*)X-Beta-Enabled=([^;]*)/);
```

Comment:
Are you sure the cookie is not httpOnly?

Reply:
It is not

https://github.com/GEWIS/sudosos-frontend/blob/03239c1e989ab4ed7791cea22c14e11ae07ecc4d/lib/nginx/default.conf#L43